CHANGELOG.rst

```diff
@@ -17,6 +17,9 @@ Added
 
 Changed
 ~~~~~~~~~
+* Pin chardet requirement #5101
```

Comment:
Can you please also include in the Changelog why this change was needed.
Following that, it might be a better fit for a Fixed section, rather then Changed.

Reply:
Thanks for feedback - fixed